Documentation/btrfs-qgroup.rst

```diff
@@ -31,6 +31,11 @@ ownership. For example a fresh snapshot shares almost all the blocks with the
 original subvolume, new writes to either subvolume will raise towards the
 exclusive limit.
 
+.. note::
+   Qgroup limit only works when qgroup is in a consistent status.
+   If by some workload, qgroup is mark inconsistent, the limit will no longer
```

Comment:
"If some workload marks qgroup as inconsistent (for example <WHAT... >) the limits will not apply until the inconsistent flag is cleared but :command:`btrfs quota rescan`.